File: MPI/mpicheck/tests/functions/2.0/MPI_Info_get_nthkey.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Info var_0;
    int var_1;
    char *var_2;
    int ret;
    /* calls */
    ret = MPI_Info_get_nthkey(var_0, var_1, var_2);
    ret = PMPI_Info_get_nthkey(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Info_get_nthkey.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       CHARACTER*(10) var_2
       INTEGER var_3
        call mpi_info_get_nthkey(var_0, var_1, var_2, var_3)
       call pmpi_info_get_nthkey(var_0, var_1, var_2, var_3)
        end program main
    